pages/views/_includes/navigation/related-links.njk

```diff
@@ -3,12 +3,20 @@
     <div class="related-card">
       <h2 class="related-card-heading">{{ title }}</h2>
       <ul class="related-card-list">
-        {% set links = functions.findItemsByName(pages, collections[collection]) %}
-        {% for page in links | released %}
+        {% if collection === 'source'%}
```

Comment:
As for right now, it is not a collection, it is just a link
Propose to separate template ("macro") for github as it is not a list of links it is a separate link block.
It definitely could reuse CSS, but it is not right to mixup related lists and single link